packages/ethers/src/client.ts

```diff
@@ -34,6 +34,8 @@ import {
 } from '@web3modal/scaffold-utils/ethers'
 import type { EthereumProviderOptions } from '@walletconnect/ethereum-provider'
 import type { Eip1193Provider } from 'ethers'
+import { W3mFrameProvider } from '@web3modal/smart-account'
+import type { CombinedProvider } from 'packages/scaffold-utils/dist/types/src/ethers'
```

Comment:
`CombinedProvider` should be available on `@web3modal/scafold-utils` instead of dirrect import